openwisp_radius/radclient/client.py

```diff
@@ -25,6 +21,19 @@ def _EncodeKeyValues(self, key, values):
             return (key, values)
         return super()._EncodeKeyValues(key, values)
 
+    def AddAttribute(self, key, value):
+        try:
+            return super().AddAttribute(key, value)
+        except KeyError:
+            # Skip attributes not found in the dictionary
+            logger.warning(f"RADIUS attribute '{key}' not found in dictionary")
```

Comment:
Are you sure this shouldn't be logged as error?

Do we need to be notified if this happens?

Reply:
This ignores checks like `Simultaneous-Use` which are not supposed to be sent to the NAS.